python_modules/libraries/dagster-k8s/dagster_k8s/executor.py

```diff
@@ -133,6 +139,7 @@ def k8s_job_executor(init_context: InitExecutorContext) -> Executor:
         namespace=exc_cfg.get("job_namespace"),  # type: ignore
         resources=exc_cfg.get("resources"),  # type: ignore
         scheduler_name=exc_cfg.get("scheduler_name"),  # type: ignore
+        run_k8s_config=exc_cfg.get("step_k8s_config"),  # type: ignore
```

Comment:
I think this deserves a comment since the `run_k8s_config` -> `step_k8s_config` mapping is subtle and this depends on the not necessarily intuitive behavior that we apply the "run" level config to all the steps